newIDE/electron-app/app/ModalWindow.js

```diff
@@ -36,6 +36,10 @@ const loadModalWindow = ({
     backgroundColor,
     modal: true,
     center: true,
+    // Hide top bar to hide close button (needed on Linux) so that
+    // one cannot close the modal window without using Save or Cancel
+    // buttons (and lose work on Piskel for instance) (Workaround for #4245).
```

Comment:
The ModalWindow is also used for yarn and jfxr.
Are we sure this removal of the top bar doesn't affect them?

Reply:
It will affect them and for the same logic: we don't want to exit those modal windows without using the `Save` or `Cancel` buttons

Reply:
If we're happy it doesn't prevent closing the window on all 3 tools, then ok 👍